frontend/assets/style/_icons.scss

```diff
@@ -81,8 +80,7 @@ $icons: (
     url("../fonts/Icons/fa-regular-400.eot?#iefix") format("embedded-opentype"),
     url("../fonts/Icons/fa-regular-400.woff2") format("woff2"),
     url("../fonts/Icons/fa-regular-400.woff") format("woff"),
-    url("../fonts/Icons/fa-regular-400.ttf") format("truetype"),
-    url("../fonts/Icons/fa-regular-400.svg#fontawesome") format("svg");
```

Comment:
@snowteamer can you help me understand this change?
@pieer is this safe?

Reply:
For some reason I couldn't bundle the app with SVG fonts URLs containing a fragment (the part after the `#` sign).
I was hesitant to remove them, but after some research it turned out that even old IE browsers should be able to handle at least one of the other specified font formats, and that woff and woff2 are much more efficient anyway.

Reply:
Fantastic reasoning. Thank you @snowteamer. I'm gonna go ahead and merge your PR now. @pieer if you disagree let us know and if necessary we can re-add in another PR.

Reply:
Sounds good to me